postgres-persistence/src/main/java/com/netflix/conductor/postgres/PostgresDataSourceProvider.java

```diff
@@ -90,6 +92,9 @@ private void flywayMigrate(DataSource dataSource) {
 
         flyway.setLocations(Paths.get("db","migration_postgres").toString());
         flyway.setDataSource(dataSource);
+        flyway.setBaselineOnMigrate(true);
```

Comment:
Consider abstracting this out into configuration.

Reply:
> Consider abstracting this out into configuration.

removed.